src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -686,7 +687,7 @@ public boolean addItemStackToInventory(final ItemStack itemStackIn)
                     {
                         this.mainInventory.set(j, itemStackIn.copy());
                         (this.mainInventory.get(j)).setAnimationsToGo(5);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)